hil/cli/helper.py

```diff
@@ -1,8 +1,15 @@
 """Helper functions for the CLI live here"""
 import json
 import sys
+import os
 from prettytable import PrettyTable
 
+try:
+    HIL_TIMEOUT = int(os.getenv('HIL_TIMEOUT', 10))
+except ValueError:
+    print("HIL_TIMEOUT is not a number")
```

Comment:
> Especially if they get it and haven't set HIL_TIMEOUT to anything yet. 

If they don't set HIL_TIMEOUT to anything then it will default to 10 seconds and no error is raised. The message should be printed only when they explicitly set it to something non-integer.